src/Microsoft.PowerShell.Commands.Utility/commands/utility/New-Object.cs

```diff
@@ -10,8 +10,14 @@
 using System.Management.Automation;
 using System.Management.Automation.Internal;
 using System.Management.Automation.Language;
+#if !UNIX
+using System.Management.Automation.Security;
+#endif
 using System.Reflection;
 using System.Runtime.InteropServices;
+#if !UNIX
+using System.Threading;
+#endif
```

Comment:
Please sort order.

Reply:
The directives have been sorted alphabetically.

Do you want `using System.Management.Automation.Security` and `using System.Threading` within the same `#if !UNIX`?

Reply:
Sorry, I skipped something...